src/main/java/com/example/sinitto/sinitto/controller/SinittoController.java

```diff
@@ -1,47 +1,72 @@
 package com.example.sinitto.sinitto.controller;
 
+import com.example.sinitto.common.annotation.MemberId;
+import com.example.sinitto.guard.dto.SeniorRequest;
 import com.example.sinitto.sinitto.dto.SinittoRequest;
 import com.example.sinitto.sinitto.dto.SinittoResponse;
+import com.example.sinitto.sinitto.service.SinittoService;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.tags.Tag;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @RequestMapping("/api/sinittos")
-@Tag(name = "[미구현]마이페이지 - 시니또용", description = "시니또 관련 마이페이지 API")
+@Tag(name = "마이페이지 - 시니또용", description = "시니또 관련 마이페이지 API")
 public class SinittoController {
 
+    private final SinittoService sinittoService;
+
+    public SinittoController(SinittoService sinittoService) {
+        this.sinittoService = sinittoService;
+    }
+
     @Operation(summary = "시니또 본인 정보 조회", description = "시니또 본인의 정보를 요청한다.")
     @GetMapping
-    public ResponseEntity<SinittoResponse> getSinittoInfo() {
-        // 임시 응답
-        return ResponseEntity.ok(new SinittoResponse(null, null, null, null, null));
+    public ResponseEntity<SinittoResponse> getSinittoInfo(@MemberId Long memberId) {
+        return ResponseEntity.ok(sinittoService.readSinitto(memberId));
+    }
+
+    @Operation(summary = "계좌정보 등록", description = "시니또가 계좌정보 등록합니다.")
+    @PostMapping("/bank")
+    public ResponseEntity<String> createSenior(@MemberId Long memberId, @RequestBody SinittoRequest sinittoRequest) {
```

Comment:
```suggestion
    public ResponseEntity<String> createSinittoBankInfo(@MemberId Long memberId, @RequestBody SinittoRequest sinittoRequest) {
```
사소한 사항인데 약간의 수정 부탁드립니다! 감사합니다!

Reply:
아하 맞네요!